From 3ceb85ae35421c5bc560c7af8d0dd1c73d071ad0 Mon Sep 17 00:00:00 2001
From: Dmitry Torokhov <dmitry.torokhov@gmail.com>
Date: Sun, 7 Mar 2010 12:37:17 -0800
Subject: USB: remove unused defintion of struct usb_device_status
Patch-mainline: v2.6.34
Git-commit: 3ceb85ae35421c5bc560c7af8d0dd1c73d071ad0

The recent rework of /proc/bus/usb/devices polling support made
this structure unused so let's remove it.

Signed-off-by: Dmitry Torokhov <dtor@mail.ru>
Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/core/devices.c |    5 -----
 1 file changed, 5 deletions(-)

--- a/drivers/usb/core/devices.c
+++ b/drivers/usb/core/devices.c
@@ -133,11 +133,6 @@ static struct device_connect_event {
 	.wait = __WAIT_QUEUE_HEAD_INITIALIZER(device_event.wait)
 };
 
-/* this struct stores the poll state for <mountpoint>/devices pollers */
-struct usb_device_status {
-	unsigned int lastev;
-};
-
 struct class_info {
 	int class;
 	char *class_name;
